docs/dev/reference/dca/config.md

```diff
@@ -29,29 +29,31 @@ $GLOBALS['TL_DCA']['tl_example']['config'] = [
+| [onsubmit_callback](../callbacks/#configonsubmit)                   | Callback functions (`array`)      | Calls a custom function after a record has been updated and passes the DataContainer object as argument.                                                                                                                |
+| [ondelete_callback](../callbacks/#configondelete)                   | Callback functions (`array`)      | Calls a custom function when a record is deleted and passes the DataContainer object as argument.                                                                                                                       |
+| [oncut_callback](../callbacks/#configoncut)                         | Callback functions (`array`)      | Calls a custom function when a record is moved and passes the DataContainer object as argument.                                                                                                                         |
+| [oncopy_callback](../callbacks/#configoncopy)                       | Callback functions (`array`)      | Calls a custom function when a record is duplicated and passes the insert ID and the DataContainer object as argument.                                                                                                  |
+| [onundo_callback](../callbacks/#configonundo)                       | Callback functions (`array`)      | Calls a custom function when a deleted record has been restored from the "undo" table.                                                                                                                                  |
+| [onversion_callback](../callbacks/#configonversion)                 | Callback functions (`array`)      | Calls a custom function when a new version of a record is created and passes the table, the insert ID and the DataContainer object as argument.                                                                         |
+| [onrestore_callback](../callbacks/#configonrestore)                 | Callback functions (`array`)      | Calls a custom function when a version of a record is restored and passes the insert ID, the table, the data array and the version as argument.  This callback is deprecated, use `onrestore_version_callback` instead. |
+| [onrestore_version_callback](../callbacks/#configonrestore_version) | Callback functions (`array`)      | Calls a custom function when a version of a record is restored and passes the table, insert ID, the table, the version and the data array as argument.                                                                  |
+| sql                                                                 | Table configuration (`array`)     | Describes table configuration, e.g. `'keys' => [ 'id' => 'primary', 'pid' => 'index' ]`                                                                                                                                 |
```

Comment:
The formatting of the table is not really that important :). Please only change the necessary line, otherwise it is hard to see what has been changed.

Reply:
I updated this pr so that only the changed lines are added. Feel free to reformat the table later on. :)